Build error on MinGW

**Godot version:** Git https://github.com/godotengine/godot/commit/db55d8a4b6ad6c14f2131e02b50689eb8380276f

**OS/device including version:** Fedora 28 on GitLab CI

**Issue description:** The build fails on MinGW, see the log below:

```
Compiling ==> thirdparty/cvtt/ConvectionKernels.cpp
thirdparty/cvtt/ConvectionKernels.cpp: In member function 'void cvtt::Internal::IndexSelectorHDR<TVectorSize>::InitHDR(int, bool, bool, const float*)':
thirdparty/cvtt/ConvectionKernels.cpp:3157:46: error: 'MakeBoolInt16' is not a member of 'cvtt::ParallelMath'
m_isInverted = ParallelMath::MakeBoolInt16(false);
^~~~~~~~~~~~~
thirdparty/cvtt/ConvectionKernels.cpp: In static member function 'static void cvtt::Internal::BC7Computer::TrySinglePlane(uint32_t, const MUInt15 (*)[4], const MFloat (*)[4], const float*, int, int, cvtt::Internal::BC7Computer::WorkInfo&, const cvtt::ParallelMath::RoundTowardNearestForScope*)':
thirdparty/cvtt/ConvectionKernels.cpp:3820:76: error: 'MakeBoolInt16' is not a member of 'cvtt::ParallelMath'
ParallelMath::Int16CompFlag isPunchThrough = ParallelMath::MakeBoolInt16(true);
^~~~~~~~~~~~~
thirdparty/cvtt/ConvectionKernels.cpp:4047:72: error: 'MakeBoolInt16' is not a member of 'cvtt::ParallelMath'
punchThroughInvalid[pIter] = ParallelMath::MakeBoolInt16(false);
^~~~~~~~~~~~~
thirdparty/cvtt/ConvectionKernels.cpp: In static member function 'static void cvtt::Internal::BC6HComputer::EvaluatePartitionedLegality(const MAInt16 (*)[3], const MAInt16 (*)[3], int, const int*, bool, cvtt::Internal::BC6HComputer::MAInt16 (*)[2][3], cvtt::ParallelMath::Int16CompFlag&)':
thirdparty/cvtt/ConvectionKernels.cpp:5315:70: error: 'MakeBoolInt16' is not a member of 'cvtt::ParallelMath'
ParallelMath::Int16CompFlag allLegal = ParallelMath::MakeBoolInt16(true);
^~~~~~~~~~~~~
thirdparty/cvtt/ConvectionKernels.cpp: In static member function 'static void cvtt::Internal::BC6HComputer::EvaluateSingleLegality(const MAInt16 (*)[3], int, const int*, bool, cvtt::Internal::BC6HComputer::MAInt16 (*)[3], cvtt::ParallelMath::Int16CompFlag&)':
thirdparty/cvtt/ConvectionKernels.cpp:5356:70: error: 'MakeBoolInt16' is not a member of 'cvtt::ParallelMath'
ParallelMath::Int16CompFlag allLegal = ParallelMath::MakeBoolInt16(true);
^~~~~~~~~~~~~
thirdparty/cvtt/ConvectionKernels.cpp: In static member function 'static void cvtt::Internal::BC6HComputer::Pack(uint32_t, const cvtt::PixelBlockF16*, uint8_t*, const float*, bool, int, int)':
thirdparty/cvtt/ConvectionKernels.cpp:5571:97: error: 'MakeBoolInt16' is not a member of 'cvtt::ParallelMath'
ParallelMath::Int16CompFlag anySame = ParallelMath::MakeBoolInt16(false);
^~~~~~~~~~~~~
thirdparty/cvtt/ConvectionKernels.cpp:5577:98: error: 'MakeBoolInt16' is not a member of 'cvtt::ParallelMath'
ParallelMath::Int16CompFlag same = ParallelMath::MakeBoolInt16(true);
^~~~~~~~~~~~~
scons: *** [thirdparty/cvtt/ConvectionKernels.windows.opt.tools.32.o] Error 1
scons: building terminated because of errors.
```

I'm unable to reproduce this issue on my local system, so I don't know what can be done about this.

**Steps to reproduce:** Compile Godot for Windows from Linux using MinGW.

--- a/thirdparty/cvtt/ConvectionKernels.cpp
+++ b/thirdparty/cvtt/ConvectionKernels.cpp
@@ -1420,6 +1420,16 @@ namespace cvtt
             return v;
         }
 
+        static bool MakeBoolInt16(bool b)
+        {
+            return b;
+        }
+
+        static bool MakeBoolFloat(bool b)
+        {
+            return b;
+        }
+
         static bool AndNot(bool a, bool b)
         {
             return a && !b;

